source/blog/2018-10-12-the-ember-times-issue-68.md

```diff
@@ -52,8 +52,17 @@ responsive: true
 
 ---
 
-## [SECTION TITLE](#section-url)
+## [Module Unification is in the final stretch!](https://github.com/emberjs/ember.js/issues/16373)
```

Comment:
this is an awesome writeup! can you change to title case re: https://github.com/emberjs/website/blob/master/source/blog/CONTRIBUTING.md#writing-style-guide? check out https://titlecaseconverter.com/

Reply:
hmm, it did `Module Unification Is in the Final Stretch`
wouldn't `Module Unification is in the Final Stretch` be more correct? 
(lowercase `is`)

Reply:
I would say just go with what it says to be consistent, up to you though, nbd!